include/cute/atom/copy_traits_xe.hpp

```diff
@@ -968,9 +997,11 @@ struct Copy_Traits<XE_2D_U16x4x32_LD_N::PREFETCH, args_t...>
 };
 
 template <class... args_t>
-struct Copy_Traits<XE_2D_U16x8x32_LD_N, args_t...>
+struct Copy_Traits_<XE_2D_U16x8x32_LD_N, args_t...>
     : XE_2D_LD_Unpack<XE_2D_U16x8x32_LD_N, args_t...> {
   using ThrID = Layout<_16>;
+
+  using BlockShape = Shape<_8, _32>;
```

Comment:
Why do just a few Copy_Traits_ have BlockShape defined here?

Reply:
Yeah good catch - missed these when reverting my refactor! I have gotten rid of them.